build/DependencyVersions.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <DotNetCoreSdkLKGVersion>3.0.100-preview-009718</DotNetCoreSdkLKGVersion>
-    <DotnetCliInternalVersion>3.0.100-preview-20181105-1</DotnetCliInternalVersion>
-    <MicrosoftNETSdkPackageVersion>3.0.100-preview.18554.1</MicrosoftNETSdkPackageVersion>
+    <DotnetCliInternalVersion>3.0.100-preview.18565.3</DotnetCliInternalVersion>
```

Comment:
@livarcocc is this correct?  Seems like two different formats, but `3.0.100-preview.18565.3` was the pushed package version from the official build.

Reply:
That's correct. We are now using arcade versioning on the CLI.

Reply:
Ok, that's what I thought, but I was under the impression we had inserted a CLI since switching to Arcade, so I was confused.